client/src/lib/components/OrganizationCard/OrganizationCard.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  export let name: string;
+  export let src: string;
+  export let status: string;
```

Comment:
@Volubyl Ici on pourrait définir un type pour éviter des erreurs

```suggestion
  export let status: "catalog" | "registered" | "pending";
```

Et je renommerais `catalog` en qqc de peut-être + clair, comme par exemple `registered_with_catalog` ? En effet il semble que ce statut fasse référence au fait que non seulement l'orga est enregistrée, mais en plus elle a un catalogue.

Reply:
Sachant qu'il ne peut pas y avoir de catalogue sans enregistrement je me suis dit que c'était plus simple et plus court.